bedrock/firefox/templates/firefox/hello/index-b.html

```diff
@@ -2,7 +2,7 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
 
-{% extends "firefox/hello/index-2016.html" %}
+{% extends "firefox/hello/index.html" %}
 
 {% set_lang_files "firefox/hello/index-2016" %}
```

Comment:
I guess referencing 2016 here is appropriate as it contains the new copy, correct?